### PR TITLE
Update EditScanViewController.swift

### PR DESCRIPTION
**Pull request by hakan-codeway**
Resolving an issue that toggling to enhanced scan on ReviewViewController causes unwanted rotation change on captured image.

### CODE REVIEW

File: WeScan/Edit/EditScanViewController.swift [hunk 1]
```diff
@@ -163,7 +163,7 @@ final class EditScanViewController: UIViewController {
         
         let croppedImage = UIImage.from(ciImage: filteredImage)
         // Enhanced Image
-        let enhancedImage = filteredImage.applyingAdaptiveThreshold()?.withFixedOrientation()
+        let enhancedImage = filteredImage.applyingAdaptiveThreshold()
```
Comment (AvdLee) [inline WeScan/Edit/EditScanViewController.swift:166]: @Boris-Em do you know for which scenario we've added the fixed orientation method? I feel like this is potentially breaking something.

Comment (hakan-codeway) [reply WeScan/Edit/EditScanViewController.swift:166]: @AvdLee I felt the same way!
However I doubled check but could not be able to trace any bugs.
I thought that it probably done as a feature however originalImage and croppedImage left out by mistake.

Comment (AvdLee) [reply WeScan/Edit/EditScanViewController.swift:166]: Looping in @julianschiavo as he merged in the [PR](https://github.com/WeTransfer/WeScan/pull/80) containing the changes. Do you remember what cases we were solving?

Comment (Boris-Em) [reply WeScan/Edit/EditScanViewController.swift:166]: I can't remember why we did this specifically, but we had to work around quite a bit of orientation issues at some point.

**Pull request closed without merging**